tools/clang/test/DXC/embed_dbg.test

```diff
@@ -1,8 +1,12 @@
 // Embed debug info
+
+// NOTE: cd into unique directory to avoid ls *.pdb affected by pdb files generated by other tests.
+// RUN: mkdir -p %T/embed_dbg && cd %T/embed_dbg
```

Comment:
Are we sure the `cd %T/embed_dbg` works for subsequent commands in this test and doesn't change CWD for any other tests?  And that this works correctly across platforms?

Reply:
Each test has separate ShellEnvironment.
cd will not execute the shell command, just update the ShellEnvironment for current test.
So it should work for all platforms.